CMakeLists.txt

```diff
@@ -495,7 +495,7 @@ target_sources(
           $<$<STREQUAL:${FILAMENT_SENSOR},YES>:src/common/filament_sensor_photoeletric.cpp>
           $<$<BOOL:${FILAMENT_SENSOR}>:src/common/thread_measurement.cpp>
           src/common/hardware_serial.cpp
-          src/common/diag.cpp
+          # src/common/diag.cpp
```

Comment:
No commented lines. Please remove it instead ☺️

Reply:
Fixed